taskcat/stacker.py

```diff
@@ -1605,10 +1616,12 @@ def define_tests(self, yamlc, test):
                     # Enforce strict json syntax
                     if self._strict_syntax_json:
                         self.check_json(cfntemplate, quite=True, strict=True)
+                    self.template_data = json.loads(cfntemplate)
```

Comment:
as json is a subset of yaml, why do we need this ? can't we just do a yaml.load on json templates ?

Reply:
Just tested this out on a sample json file. Theoretically yes. However, other functions within stacker.py take this approach. From my perspective, that was done for a reason, and introducing a change to the standard (of the module) is counterintuitive.